### ISSUE TITLE
Inductee Points Rollover

### ISSUE THREAD

**Issue by MeghajV** [HKN-UCSD/hkn-portal#63]
Inductees that rollover to a new induction cycle should - 
1. Keep their points if the rollover was in the same school year
2. Keep a max of 3 general points if the rollover was over a school year

Something to consider with this is people that were guests in a previous cycle and become inductees in the next cycle should not retain their points from when they were guests. 

There are many potential ways to handle this but this is blocked by inductee classes

**Comment by MeghajV** [comment]
Duplicate of #41
